go.mod

```diff
@@ -23,7 +23,7 @@ require (
 
 replace (
 	github.com/Microsoft/hcsshim => github.com/projectcalico/hcsshim v0.8.9-calico
-	github.com/kelseyhightower/confd => github.com/projectcalico/confd v1.0.1-0.20210706171510-82673c0a6e04
+	github.com/kelseyhightower/confd => github.com/projectcalico/confd v1.0.1-0.20210715022422-363d56715100
```

Comment:
I would expect this to match https://github.com/projectcalico/confd/commit/788346b7b46716fda8cff8fdf8ccc4f60a26a593

Reply:
Sorry thought this was supposed to be against master, this is correct.